Dockerfile.windows

```diff
@@ -249,19 +252,60 @@ RUN `
   Remove-Item C:\binutils.zip; `
   Remove-Item C:\gitsetup.zip; `
   `
-  Write-Host INFO: Creating source directory...; `
-  New-Item -ItemType Directory -Path ${GOPATH}\src\github.com\docker\docker | Out-Null; `
+  # Ensure all directories exist that we will require below....
+  $srcDir = """$Env:GOPATH`\src\github.com\docker\docker\bundles"""; `
+  Write-Host INFO: Ensuring existence of directory $srcDir...; `
+  New-Item -Force -ItemType Directory -Path $srcDir | Out-Null; `
   `
   Write-Host INFO: Configuring git core.autocrlf...; `
-  C:\git\cmd\git config --global core.autocrlf true; `
+  C:\git\cmd\git config --global core.autocrlf true; 
+
+  # Need a different layer  because the current layer  will not
+  # have go in its path. We could always figure that out either
+  # by observing  the current  behavior of  go-s installer (and 
```

Comment:
I'm a bit confused by this, because Go doesn't have an installer; we're downloading a zip and extract it to `C:\go`. We update the path ourselves at this line; https://github.com/moby/moby/blob/c057825cf56850ffb97cae532d0bfa261b4b9a53/Dockerfile.windows#L211

Does Windows only evaluate `PATH` when the shell is started, and not when updated in the same shell?

Reply:
That is correct[confirmed the behavior from a Powershell command prompt]. The below works - or just doing it in another layer, which is what I chose to do. 

`  $env:Path += """;c:\go\bin"""; `
`  $env:Path += """;c:\git\cmd"""; `

My A/B testing did not produce a clear "winner" so it looks like we are good either way.